config/application.rb

```diff
@@ -1,73 +1,28 @@
-
-    # Only load the plugins named here, in the order given (default is alphabetical).
-    # :all can be used as a placeholder for all plugins not explicitly named.
-    # config.plugins = [ :exception_notification, :ssl_requirement, :all ]
-
-    # Activate observers that should always be running.
-    # config.active_record.observers = :cacher, :garbage_collector, :forum_observer
-
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
-
-    # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
-    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
-
-    # Configure the default encoding used in templates for Ruby 1.9.
-    config.encoding = "utf-8"
-
-    # Configure sensitive parameters which will be filtered from the log file.
-    config.filter_parameters += [:password, :name, :email, :email_confirmation, :textdetails, :what_doing, :what_wrong]
+    config.action_dispatch.rack_cache = nil
 
-    # Enable escaping HTML in JSON.
     config.active_support.escape_html_entities_in_json = true
 
-    # Use SQL instead of Active Record's schema dumper when creating the database.
-    # This is necessary if your schema can't be completely dumped by the schema dumper,
-    # like if you have constraints or database-specific column types
-    # config.active_record.schema_format = :sql
-
-    # Enforce whitelist mode for mass assignment.
-    # This will create an empty whitelist of attributes available for mass-assignment for all models
-    # in your app. As such, your models will need to explicitly whitelist or blacklist accessible
-    # parameters by using an attr_accessible or attr_protected declaration.
-    # config.active_record.whitelist_attributes = true
-
-    # Enable the asset pipeline
     config.assets.enabled = true
-
-    # Version of your assets, change this if you want to expire all your assets
+    config.assets.precompile += %w(feedback.js feedback.css feedback-ie6.css)
+    config.assets.prefix = '/feedback' # this has to match the path configured in puppet and deploy scripts.
     config.assets.version = '1.0'
 
-    config.assets.prefix = '/feedback' # this has to match the path configured in puppet and deploy scripts.
+    config.eager_load_paths += %W(#{config.root}/lib)
 
-    # Precompile additional assets (application.js, application.css, and all non-JS/CSS are already added)
-    config.assets.precompile += %w(feedback.js feedback.css feedback-ie6.css)
+    config.encoding = "utf-8"
 
-    # Disable Rack::Cache
-    config.action_dispatch.rack_cache = nil
+    config.filter_parameters += [:password, :name, :email, :email_confirmation, :textdetails, :what_doing, :what_wrong]
+
+    config.load_defaults 5.1
```

Comment:
Would this overwrite settings above? Maybe move it up to be sure.

Reply:
Good point - i'll put it at the top